applications/ContactStructuralMechanicsApplication/custom_conditions/ALM_frictionless_mortar_contact_condition.cpp

```diff
@@ -30442,10 +30442,10 @@ bounded_matrix<double, 10, 10> AugmentedLagrangianMethodFrictionlessMortarContac
     const bounded_matrix<double, 2, 2> X1 = rDerivativeData.X1;
     const bounded_matrix<double, 2, 2> X2 = rDerivativeData.X2;
     
-    const array_1d<double, 2> LMNormal = ContactUtilities::GetVariableVector<2>(this->GetGeometry(), NORMAL_CONTACT_STRESS, 0);
+    const array_1d<double, 2> LMNormal = MortarUtilities::GetVariableVector<2>(this->GetGeometry(), NORMAL_CONTACT_STRESS, 0);
```

Comment:
Do you need a copy? A cost reference is not enough?

Reply:
What do you mean?